dataprep/data_connector/schema.json

```diff
@@ -8,7 +8,7 @@
         "request",
         "response"
     ],
-    "additionalProperties": false,
+    "additionalProperties": true,
```

Comment:
If you'd like to add new fields to the config file, please write it explicitly. `additionalProperties: true` is not a good practice.